drafts/use-cases/3.obtaining-events.md

```diff
@@ -0,0 +1,110 @@
+## Entry point
+
+### Story
+As an application user
+I want to display all my events
+So I can find out what awaits me in the nearest future.
```

Comment:
I would also keep this generic

As an API consumer
I want to be able to retrieve all members of a collection
So that I can process them further.